docs/concepts/istio.md

```diff
@@ -0,0 +1,26 @@
+# Istio
```

Comment:
Can you move the docs to the `docs/usage` folder? Can you also add links to them in the index (`docs/README.md`).